storage_controller/src/main.rs

```diff
@@ -213,6 +213,13 @@ struct Cli {
     /// This option exists primarily for testing purposes.
     #[arg(long, default_value = "3", value_parser = clap::value_parser!(i64).range(1..))]
     timeline_safekeeper_count: i64,
+
+    /// When set, actively checks and initiates heatmap downloads/uploads during reconciliation.
+    /// This speed up migrations by avoiding the default wait for the heatmap download interval.
+    /// Primarily useful for testing to reduce test execution time.
+    #[cfg(feature = "testing")]
+    #[arg(long, default_value = "true")]
```

Comment:
It won't work. The only value you can specify is `true` by `--kick-secondary-downloads`. And if missing, the default is also `true`, so no way to set `false`

Here is discussion how you can overcome it: https://neondb.slack.com/archives/C0277TKAJCA/p1740669723609099 with `--kik-secondary-downloads=false`

Reply:
By the way, are you sure we need this option at all?

It's now on even in your new test, and the test passes